assets/styles/global.scss

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
I haven't really thought about styling/looks in this PR at all, this is something I hope to focus more on once we have some content for the pages.